src/settings.ts

```diff
@@ -1,3 +1,4 @@
+import { QualifiedConfig } from '@commitlint/types/lib/load';
```

Comment:
This type is available in the root package:

```suggestion
import { QualifiedConfig } from '@commitlint/types';
```